applications/DEM_application/test_examples/basic_benchmarks/BenchTemp.info

```diff
@@ -0,0 +1,11246 @@
+            Parallelism Info:  MPI number of nodes: 1
+            Parallelism Info:  OMP number of processors: 1
+          **************************************************
+
+Initialization Complete
+
+Main loop starting...
+Total number of TIME STEPs expected in the calculation: 13888
+
+Computing points in the curve... 9 point(s) left to finish.... 
+
+The Id 1 was found in the model part
+Finalizing execution...
+running CleanUpOperations
+Calculation ends at instant: 1509634262.3653219
+Calculation ends at processing time instant: 26.789479
+Elapsed processing time: 2.8836559999999984
+Elapsed real time: 2.8843610286712646
+ANALYSIS COMPLETED
+
+  [Reading Nodes    : 1 nodes read]
+  [Reading Elements : 1 elements read] [Type: SphericParticle3D]
+  [Total Lines Read : 49]
+  [Reading Nodes    : 4 nodes read]
+  [Reading Conditions : 2 conditions read] [Type: RigidFace3D3N]
+  [Total Lines Read : 54]
+  [Total Lines Read : 1]
+  [Total Lines Read : 5]
+
+Global Damping parameter not found! No damping will be applied...
+
+DOFs for the DEM solution added correctly
+DOFs for the DEM solution added correctly
+DOFs for the DEM solution added correctly
+
+Initializing Problem...
+
+Assigning DEM_D_Hertz_viscous_Coulomb to Properties 1
+
+Assigning ForwardEulerScheme to properties 1
+          **************************************************
+            Parallelism Info:  MPI number of nodes: 1
+            Parallelism Info:  OMP number of processors: 1
+          **************************************************
+
+Initialization Complete
+
+Main loop starting...
+Total number of TIME STEPs expected in the calculation: 13888
+
+Computing points in the curve... 8 point(s) left to finish.... 
+
+The Id 1 was found in the model part
+Finalizing execution...
+running CleanUpOperations
+Calculation ends at instant: 1509634265.2935488
+Calculation ends at processing time instant: 29.715909
+Elapsed processing time: 2.9227719999999984
+Elapsed real time: 2.924567937850952
+ANALYSIS COMPLETED
+
+  [Reading Nodes    : 1 nodes read]
+  [Reading Elements : 1 elements read] [Type: SphericParticle3D]
+  [Total Lines Read : 49]
+  [Reading Nodes    : 4 nodes read]
+  [Reading Conditions : 2 conditions read] [Type: RigidFace3D3N]
+  [Total Lines Read : 54]
+  [Total Lines Read : 1]
+  [Total Lines Read : 5]
+
+Global Damping parameter not found! No damping will be applied...
+
+DOFs for the DEM solution added correctly
+DOFs for the DEM solution added correctly
+DOFs for the DEM solution added correctly
+
+Initializing Problem...
+
+Assigning DEM_D_Hertz_viscous_Coulomb to Properties 1
+
+Assigning ForwardEulerScheme to properties 1
+          **************************************************
+            Parallelism Info:  MPI number of nodes: 1
+            Parallelism Info:  OMP number of processors: 1
+          **************************************************
+
+Initialization Complete
+
+Main loop starting...
+Total number of TIME S
```

Comment:
Is this file really needed?

Reply:
it is not.

Reply:
I have removed the file. Sorry, it slipped.